setup.py

```diff
@@ -33,6 +33,9 @@
     )
 
 
+csplayout_requirements = [
+    # TODO: Move `python-constraint` from core requirements to here for Terra 0.21.
```

Comment:
Do you want to have python constraint duplicated here too, it shouldn't hurt anything pip will just resolve it to the same package

Reply:
Yeah, that seems sensible.  Done in 9e865bddb.